docs/admin/env_variables.md

```diff
@@ -35,6 +35,8 @@ Environment variables in this section only apply to the server side.
 |----------------------|-----------|
 |RDB\_ELECTION\_TIMEOUT|Raft election timeout used by RDBs in milliseconds. INTEGER. Default to 7000 ms.|
 |RDB\_REQUEST\_TIMEOUT |Raft request timeout used by RDBs in milliseconds. INTEGER. Default to 3000 ms.|
+|RDB_LEASE_MAINTENANCE_GRACE|Raft grace period of leadership lease maintenance used by RDBs in milliseconds. INTEGER. Default to 7000 ms. If a Raft leader is unable to maintain leadership leases from a majority for more than RDB_ELECTION_TIMEOUT + RDB_LEASE_MAINTENANCE_GRACE, it steps down voluntarily.|
+|RDB_USE_LEASES|Whether RDBs shall use Raft leadership leases, instead of RPCs, to verify leadership. BOOL. Default to true. Rafts track leadership leases regardless; this environment variable essentially controls whether RDBs use Raft leadership leases to improve RDB TX performance.|
```

Comment:
very minor suggestion (only if something else needs changing)
"Rafts track" --> "Raft tracks" (unless I am reading it differently than you intended).

Reply:
Ah yes, I see. The wording is good, and it is not too awkward. This same thought crossed my mind while reading the text, that it may refer to all of the instances.

Reply:
OK, I'll keep the wording this way for now. Won't mind changing it to singular form in the future if that reads better.